content/news/2020-12-01-about-me.md

```diff
@@ -0,0 +1,116 @@
+title: About my Outreachy internship with Mixxx DJ: Intro
```

Comment:
As I explained before, titles with colons now need to be I quotes, e. G:

```suggestion
title: "About my Outreachy internship with Mixxx DJ: Intro" 
```

Reply:
ohhh okay. I get it now. Thanks.